ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -535,37 +535,38 @@ const CompletionBanner = ({
         <div style={styles.completedText}>
           Researcher Workbench data access is complete.
         </div>
-        {enableInitialCreditsExpiration && (
-          <>
-            <div style={styles.completedText}>
-              Your credits expire on{' '}
-              {displayDateWithoutHours(
-                profile.initialCreditsExpirationEpochMillis
-              )}
-              .
-            </div>
-            <div style={styles.completedText}>
-              (You have {initialCreditsValidityPeriodDays} days to use credit
-              after gaining data access. You may request an extension when it
-              gets closer to your credit expiration date, which will extend your
-              credit expiration date to a total of{' '}
-              {initialCreditsExtensionPeriodDays} days from the day you gained
-              data access.)
-            </div>
-            <div style={styles.completedText}>
-              Learn more{' '}
-              <LinkButton
-                style={{ display: 'inline' }}
-                onClick={() =>
-                  window.open(supportUrls.initialCredits, '_blank')
-                }
-              >
-                here
-              </LinkButton>
-              .
-            </div>
-          </>
-        )}
+        {enableInitialCreditsExpiration &&
+          profile.initialCreditsExpirationEpochMillis && (
```

Comment:
When a user is bypassed either individually or institutionally, they will have a null initialCreditsExpirationEpochMillis value. The profile does have a initialCreditsExpirationBypassed field, but that only represents individual bypass and not institutionally.